plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -246,10 +244,31 @@ def inspect(filter, options = {})
       ensure
         helper.remove_helper
       end
-      scope.extracted = false
 
+      scope.extracted = !!do_extract
```

Comment:
This is a confusing statement. Maybe it would be clearer to put a `scope.extracted = true` at the end of the extraction code and a `scope.extracted = false` at the end of the code, which doesn't extract.